### ISSUE TITLE
Unhandled Promise Rejection warning

### ISSUE THREAD

**Issue by muneeburrahman** [jshemas/openGraphScraper#33]
I am trying to fetch data for a number of links. It is giving the Unhandled Promise Rejection warning .
Following is the code

     ```
        for(var i=0; i< 100;i++){
		var options = {'url': url};
	  ogs(options, function (err, results) {
				if(err){	
                               console.log('error')
				}
				else{
                                 console.log(results);
				}
	  });
  }
```

**Comment by jshemas** [comment]
I just put in a fix for this, please update to version `2.4.0`

**Comment by muneeburrahman** [comment]
Resolved . Thank you
